hplsql/src/main/java/org/apache/hive/hplsql/objects/TableClass.java

```diff
@@ -136,4 +155,8 @@ public MethodDictionary methodDictionary() {
   public boolean rowType() {
     return rowType;
   }
+
+  public static boolean isTableAttributeExists(String property) {
+    return TABLE_ATTRIBUTES_LIST.contains(property);
```

Comment:
TABLE_ATTRIBUTES_LIST is more like a Set<String>.

Can `methodDictionary.containsKey(property)` be used instead?

Reply:
Changed the code accordingly.